src/main/java/teammates/ui/controller/FileDownloadResult.java

```diff
@@ -69,5 +51,9 @@ public String getFileName() {
     public String getFileContent() {
         return this.fileContent;
     }
+    
+    public String getFileType(){
```

Comment:
No whitespace between `()` and `{` here, causing checkstyle violations. Should be possible to review this once it is fixed and the build passes.